GOVERNANCE.md

```diff
@@ -0,0 +1,131 @@
+
+- commitment to the project:
+  - participate in discussions, contributions, code and documentation reviews
+      for [TODO: Time Period] or more,
+  - perform reviews for [TODO:Number] non-trivial pull requests,
+  - contribute [TODO:Number] non-trivial pull requests and have them merged,
+- ability to write quality code and/or documentation,
+- ability to collaborate with the team,
+- understanding of how the team works (policies, processes for testing and code review, etc),
+- understanding of the project's code base and coding and documentation style.
+  <!-- add any additional Maintainer requirements here -->
+
+A new Maintainer must be proposed by a PR to the [MAINTAINERS](./MAINTAINERS) file.
+A simple majority vote of existing Maintainers approves the application.
+
+Maintainers who are selected will be granted the necessary GitHub rights,
+and invited to the [private maintainer mailing list][mailing-list].
+
+## Meetings
+
+Time zones permitting, Maintainers are expected to participate in the regular
+[public developer meeting][meeting]).
+
+Maintainers will also have closed meetings in order to discuss security reports
+or Code of Conduct violations.  Such meetings should be scheduled by any
+Maintainer on receipt of a security issue or CoC report.  All current Maintainers
+must be invited to such closed meetings, except for any Maintainer who is
+accused of a CoC violation.
+
+## CNCF Resources
+
+Any Maintainer may suggest a request for CNCF resources, either in the
+[mailing list](TODO: link to developer/maintainer mailing list), or during a
+meeting.  A simple majority of Maintainers approves the request.  The Maintainers
+may also choose to delegate working with the CNCF to non-Maintainer community
+members.
+
+## Code of Conduct
+
+<!-- This assumes that your project does not have a separate Code of Conduct
+Committee; most maintainer-run projects do not.  Remember to place a link
+to the private Maintainer mailing list or alias in the code-of-conduct file.-->
+
+[Code of Conduct](./CODE_OF_CONDUCT.md) violations by community members will be
+discussed and resolved on the [private Maintainer mailing list][mailing-list].
+If the reported CoC violator is a Maintainer, the Maintainers will instead
+designate two Maintainers to work with CNCF staff in resolving the report.
+
+## Voting
+
+While most business in Kured is conducted by "lazy consensus", periodically
+the Maintainers may need to vote on specific actions or changes.
+A vote can be taken on [the developer mailing list](TODO) or
+[the private Maintainer mailing list][mailing-list] for security or conduct
+matters. Votes may also be taken at [the developer meeting][meeting]. Any
+Maintainer may demand a vote be taken.
+
+Most votes require a simple majority of all Maintainers to succeed. Maintainers
+can be removed by a 2/3 majority vote of all Maintainers, and changes to this
+Governance require a 2/3 vote of all Maintainers.
```

Comment:
The Flux project chose to go with https://fluxcd.io/governance/#unanimity-decisions for "big decision" kind of things. What I like about it is:
- aiming for consensus by not allowing vetos
- setting a deadline so decisions are not blocked on people being on extended holidays, etc

Reply:
Process-wise we could issues in `/kubereboot/kured` for documenting vote-worthy things. WDYT?

Reply:
I like this approach. The difficulty comes if defining the boundary between "big decisions" and "decisions". Given that the document as a whole describes expected community behaviors that include trust and cooperation, I think we shouldn't have to worry about a strict definition, we can rely upon the "good vibes" part of this doc to be the foundation of the community, and the "strict rules" part secondary.

So +1 to your suggestions.

Reply:
If you check out https://fluxcd.io/governance/ a lot of "big decision" type stuff is spelled out. Almost all of it in the realms of governance. If we are talking about big code changes, we could adopt a lighter version of https://github.com/fluxcd/community/blob/main/PROCESS.md#proposal-process?

But all of this might as well just be added in the second iteration of governance?

Reply:
+1 to formalizing this definition in a future version, if in practice the community will benefit from that. In the meantime it's worth leaning on common sense as an initial guidepost (IMO).